ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala

```diff
@@ -179,6 +179,14 @@ class ServicesMetrics(override val prefix: MetricName, override val registry: Me
     )
     val writeOperationForDocs: Timer = Timer(prefix :+ "<operation>", null)
 
+    @MetricDoc.Tag(
+      summary = "The time to execute a write service operation.",
+      description = """The write service is an internal interface for changing the state through
+                      |the synchronization services. The methods in this interface are all methods
+                      |that are supported uniformly across all ledger implementations. This metric
+                      |exposes the time needed to execute each operation.""",
+      qualification = Traffic,
+    )
```

Comment:
1. The text above seems to be the general text that accompanies all <operations>. It needs to be slightly more adopted to the `submit_transaction`
2. Actually it is only the _count portion of it that is a Traffic metric, I would suggest to do 
`val submitOperationForDocs: Timer = Timer(prefix :+ "submit_transaction_count", null)`

Reply:
i.e. to be more specific regarding which component of the metric we are using (mean, count)

Reply:
I thin it only affects timers used for Traffic